services/logger.js

```diff
@@ -0,0 +1,109 @@
+const createLogger = () => {
+  const isProdMode =
+    process.env.NODE_ENV === "prod" || process.env.NODE_ENV !== "development"
+      ? true
+      : false;
```

Comment:
Shouldn't this be enough?
```suggestion
  const isProdMode =
    process.env.NODE_ENV === "prod" || process.env.NODE_ENV !== "development";
```

Otherwise, it looks like `const abc = true ? true : false`

Reply:
oof, you're totally right!